### PR TITLE
Jenkins + Slang + Docker use case

### PR DESCRIPTION
**Pull request by Bonczidai**
ppt presentation sent over mail

Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

File: org/openscore/slang/lifecycle/automation/deploy_artifact.sl [hunk 1]
```diff
@@ -0,0 +1,56 @@
+namespace: org.openscore.slang.lifecycle.automation
```
Comment (meirwah) [inline org/openscore/slang/lifecycle/automation/deploy_artifact.sl]: missing doc...

File: org/openscore/slang/lifecycle/automation/deploy_artifact.sl [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
```
Comment (meirwah) [inline org/openscore/slang/lifecycle/automation/deploy_artifact.sl]: missing documentation ...

File: org/openscore/slang/lifecycle/automation/deploy_artifact_flow.sl [hunk 3]
```diff
@@ -0,0 +1,30 @@
+namespace: org.openscore.slang.lifecycle.automation
```
Comment (meirwah) [inline org/openscore/slang/lifecycle/automation/deploy_artifact_flow.sl]: missing documentation ...

File: slang_content.iml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (meirwah) [inline slang_content.iml]: don't commit .iml files!!! :-1:

File: slang-content [hunk 5]
```diff
@@ -0,0 +1 @@
+Subproject commit 92df59ef02075f76ae112133663c10a1052da46d
```
Comment (meirwah) [inline slang-content]: what is this ? @orius123 any idea?

Comment (orius123) [inline slang-content]: looks like some project merge problem

Comment (meirwah) [inline slang-content]: but merge seem green (OK) ... weird..

Comment (meirwah) [inline slang-content]: maybe missing gitignore here?

Comment (orius123) [inline slang-content]: look at commit history of the branch. last merge look weird to me.

Comment (meirwah) [inline slang-content]: @avimora any idea?

Comment (avimora) [inline slang-content]: There is a strange `slang content` folder inside the fork, may be it's related

**Comment by oritstone** [comment]
Developed during the Hackathon. 
It is a great story, but we decided not to merge it

**Pull request closed without merging**